docker/target/docker.go

```diff
@@ -41,3 +41,8 @@ func (Docker) Up(ctx context.Context) error {
 func (Docker) Down(ctx context.Context) error {
 	return docker.Docker(ctx, "compose", "down", "-v", "--remove-orphans")
 }
+
+// Restart restarts containers in daemon mode
+func (Docker) Restart(ctx context.Context) {
+	mg.SerialCtxDeps(ctx, Docker.Down, Docker.Up)
```

Comment:
If this is meant to be part of this pr there's also `docker compose restart`.

Reply:
I can move it into a separate PR if requested. Yes, `compose restart` was my initial choice, but I doesn't remove volumes like the current `Down` command.

Reply:
Roger, in that case approve from me.